cookbook/recipes/combined-listings/patches/ProductForm.tsx.8e409a.patch

```diff
@@ -0,0 +1,90 @@
+index e8616a61..838a903a 100644
+--- a/templates/skeleton/app/components/ProductForm.tsx
++++ b/templates/skeleton/app/components/ProductForm.tsx
+@@ -11,9 +11,11 @@ import type {ProductFragment} from 'storefrontapi.generated';
+ export function ProductForm({
+   productOptions,
+   selectedVariant,
++  combinedListing,
+ }: {
+   productOptions: MappedProductOptions[];
+   selectedVariant: ProductFragment['selectedOrFirstAvailableVariant'];
++  combinedListing: boolean;
+ }) {
+   const navigate = useNavigate();
+   const {open} = useAside();
+@@ -53,9 +55,10 @@ export function ProductForm({
+                       replace
```

Comment:
I think having `replace` in here is wrong. It leads to this ux issue

https://github.com/user-attachments/assets/2081b626-c1aa-4c5e-ae6f-27a4f0faf4b0